### PR TITLE
[DAWN] utils: stateEnum -> robotState, runtimeStateEnum -> runtimeState

### PR DESCRIPTION
**Pull request by kevinji**
Cherry-picked from the other branch.

### CODE REVIEW

File: dawn/main/ansible/Ansible.js [hunk 1]
```diff
@@ -98,14 +98,17 @@ ipcMain.on('NOTIFY_UPLOAD', () => {
 function buildProto(data) {
   let status = null;
   switch (data.studentCodeStatus) {
-    case stateEnum.TELEOP:
+    case robotState.TELEOP: {
```
Comment (yizow) [inline dawn/main/ansible/Ansible.js]: brackets don't match style in the entire rest of dawn, and these blocks aren't at the level where they would be useful.

**Review by yizow — CHANGES_REQUESTED** [review]
style change

**Comment by kevinji** [comment]
Fixed style issue.

**Review by yizow — APPROVED** [review]
LGTM

**Pull request merged**